File: LibraryManage/LibraryManage/Model/Books.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LibraryManage.Model
{
    public class Books
    {
        public int id {  get; set; }

        public string title { get; set; }

        public string author { get; set; }

        public string category { get; set; }

        public int quantity { get; set; }

        public string createdAt { get; set; }

        Books() { }
    }
}

File: MyServer/scr/model/books.js

import mongoose from "mongoose";

const bookTable = mongoose.Schema({
    title: {
        type: String,
        require: true,
        unique: true
    },
    author: {
        type: String,
        require: true
    },
    category: {
        type: String,
        require: true
    },
    quantity: {
        type: Number,
        require: true
    },
}, {timestamps: true, versionKey: false});

export default mongoose.model("bookModel", bookTable, "books");